canmatrix/canmatrix.py

```diff
@@ -29,12 +29,20 @@
 
 from __future__ import division
 import math
+import attr
 
 from collections import OrderedDict
 
 import logging
 import fnmatch
 
+try:
+    import decimal
+except ImportError:
```

Comment:
In what scenario do we expect `decimal` to not be importable?  It's in stdlib for Python 2.4+ and it seems all version of 3.